public/js/clients/firefox.js

```diff
@@ -57,13 +57,13 @@ function connectClient() {
     new DebuggerTransport(socket) : new WebSocketDebuggerTransport(socket);
   debuggerClient = new DebuggerClient(transport);
 
-  debuggerClient.connect().then(() => {
+  debuggerClient.connect(()=>{}, 2000).then(() => {
```

Comment:
This won't work. There were times when for some reason it would take a while to connect to firefox (say my machine is already doing something heavy, or for whatever reason) and refreshing the page a lot while developing I'd see it not connect 1/4 of the time, which is *extremely* frustrating. We need to find another way to do this, or leave it at 6 seconds.

Chrome is still second-class at this point, so I don't think we should turn it on unless it doesn't affect the Firefox workflow.